enqueue/job-queue-pack/0.8/manifest.json

```diff
@@ -0,0 +1,5 @@
+{
+    "copy-from-recipe": {
+        "etc/": "%ETC_DIR%/"
```

Comment:
This should be `"config/": "%CONFIG_DIR%/"` and the `etc/packages` dir in the recipe needs to be renamed to `config/packages`